core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -142,7 +142,8 @@
     TableStatistics getTableStatistics(Session session, TableHandle tableHandle, Constraint constraint);
 
     /**
-     * Get the names that match the specified table prefix (never null).
+     * Get the relation names that match the specified table prefix (never null).
+     * This includes all relations (e.g. tables, views, materialized views).
```

Comment:
I did not check. Is this accurate for all the connector now?

Reply:
I went through all the implementations of `io.trino.spi.connector.ConnectorMetadata#listViews` and `io.trino.spi.connector.ConnectorMetadata#listMaterializedViews` and the corresponding `listTables` implementation does indeed return the views / materialized views as well.

Reply:
> I did not check. Is this accurate for all the connector now?

even if it wasn't the case yet, we already have this
https://github.com/trinodb/trino/blob/8b0c754d9d2e6c4e5ea4eed0c8c8cefb9146fcc0/core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java#L210-L213

and Metadata just calls that method.